### PR TITLE
Consolidate the stop script to avoid stopping the kafka service

### PR DESCRIPTION
**Pull request by wenbingshen**
This PR resolves [1531](https://github.com/linkedin/cruise-control/issues/1531).

For cruise control, we should use the main class to filter the process id in the stop step, so as to avoid accidentally killing other processes that rely on cruise control.

### CODE REVIEW

File: kafka-cruise-control-stop.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # See License in the project root for license information.
 
 SIGNAL=${SIGNAL:-TERM}
-PIDS=$(ps ax | grep -i 'cruise-control' | grep java | grep -v grep | awk '{print $1}')
+PIDS=$(ps ax | grep -i 'KafkaCruiseControlMain' | grep java | grep -v grep | awk '{print $1}')
```
Comment (efeg) [inline kafka-cruise-control-stop.sh:6]: Nits:
1. Is case insensitive matching really needed? If not can we drop `-i` option?
2. According to `kafka-cruise-control-start.sh` `java` always precedes `KafkaCruiseControlMain`. Can we enforce that in our command?

 -- i.e. would the following work?
```
PIDS=$(ps ax | grep 'java.*KafkaCruiseControlMain' | grep -v grep | awk '{print $1}')
```

**Review by efeg — COMMENTED** [review]
@wenbingshen Thanks for the PR!
Left a comment.

File: kafka-cruise-control-stop.sh (hunk 1 above)
Comment (wenbingshen) [reply kafka-cruise-control-stop.sh]: Thanks for you comment. Good suggestion. I submitted your feedback.

**Review by efeg — APPROVED** [review]
@wenbingshen thanks for the update! LGTM.

**Pull request merged**